pkg/services/sqlstore/org_test.go

```diff
@@ -35,6 +35,44 @@ func TestAccountDataAccess(t *testing.T) {
 			So(len(query.Result), ShouldEqual, 3)
 		})
 
+		Convey("Given we have organizations, we can limit and paginate search", func() {
+			var err error
+			var cmd *models.CreateOrgCommand
+			ids := []int64{}
```

Comment:
What's the point of this variable, since you don't actually use it for anything?